ruby/ql/lib/codeql/ruby/ast/Literal.qll

```diff
@@ -211,11 +272,15 @@ private class FalseLiteral extends BooleanLiteral, TFalseLiteral {
 
   FalseLiteral() { this = TFalseLiteral(g) }
 
-  final override string getValueText() { result = g.getValue() }
+  final override string toString() { result = g.getValue() }
 
   final override predicate isFalse() { any() }
 }
 
+private class RequiredStringConstantValue extends RequiredConstantValue {
```

Comment:
For these and the `RequiredStringConstantValueN` variations below, it would be nice to give them a name that makes it clear what they apply to. E.g. in this case I think this should be something like `RequiredEncodingConstantValue`.